editor/script/engine/bitsy.js

```diff
@@ -234,9 +234,24 @@ function onready(startWithTitle) {
 		canvas.addEventListener('touchend', input.ontouchend, {passive:false});
 	}
 	else {
-		document.addEventListener('touchstart', input.ontouchstart, {passive:false});
-		document.addEventListener('touchmove', input.ontouchmove, {passive:false});
-		document.addEventListener('touchend', input.ontouchend, {passive:false});
+		// creates a 'touchTrigger' element that covers the entire screen and can universal have touch event listeners added w/o issue.
+
+		// we're checking for existing touchTriggers both at game start and end, so it's slightly redundant.
```

Comment:
what do you think about adding the touch trigger div as an element to the export game HTML template instead of creating it dynamically? then all the css and id stuff could live in the markup instead of in the JS code

Reply:
My thinking is that this is an accessibility issue.

Whatever few users encountered this issue never reported it, they probably bounced off of the content and never played the game, and the creator of said game never knew they were left out. If you move the trigger into the markup, and a creator not knowing what it is or does accidentally alters it when modifying their game, then suddenly that game again becomes unusable for those few users, without anyone noticing.

I don't love CSS-in-JS, but IMHO, this seems like this a good use for it.

Reply:
that makes sense to me! carry on :)